packages/manager/src/components/RegionSelect/RegionSelect.styles.ts

```diff
@@ -1,16 +1,25 @@
 import DoneIcon from '@mui/icons-material/Done';
 import { styled } from '@mui/material/styles';
 
+import { Box } from 'src/components/Box';
 import { ListItem } from 'src/components/ListItem';
 
-export const GroupHeader = styled('div', {
-  label: 'RegionSelectGroupHeader',
+export const StyledAutocompleteContainer = styled(Box, {
+  label: 'RegionSelect',
 })(({ theme }) => ({
-  color: theme.color.headline,
-  fontFamily: theme.font.bold,
-  fontSize: '1rem',
-  padding: '16px 4px 4px 10px',
-  textTransform: 'initial',
+  '& .MuiAutocomplete-groupLabel': {
+    color: theme.color.headline,
+    fontFamily: theme.font.bold,
+    fontSize: '1rem',
+    lineHeight: 1,
+    padding: '16px 4px 8px 10px',
+    textTransform: 'initial',
+  },
+  '& .MuiAutocomplete-listbox': {
+    '& li:first-child .MuiAutocomplete-groupLabel': {
```

Comment:
```suggestion
    '& li:first-of-type .MuiAutocomplete-groupLabel': {
```

![image](https://github.com/linode/manager/assets/115299789/5f9c194a-a423-453c-b6c3-619cf2d5752c)

Reply:
right on - fixed!